src/nemo_safe_synthesizer/data_processing/timeseries_validation.py

```
@@ -0,0 +1,430 @@
+    is_elapsed_time: bool,
+) -> tuple[TimeSeriesGroupTimestampStats, ...]:
+    stats_list = []
+
+    for group_name, group_df in df.groupby(group_by_col, sort=False):
+        timestamps = group_df[timestamp_col]
+        intervals = _interval_seconds(timestamps, is_elapsed_time)
+        interval = None
+        if not intervals.empty:
+            unique_intervals = intervals.unique()
+            if len(unique_intervals) == 1 or (unique_intervals.max() - unique_intervals.min()) < 0.1:
+                interval = int(round(float(intervals.iloc[0])))
```

Ruff (0.15.18)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)